tests/ui/pages/main/main-page.ts

```diff
@@ -0,0 +1,23 @@
+import InputField from '../../elements/input-field';
+import BasePage from '../base-page';
+
+const SELECTORS = {
+  searchBar: `.MuiBox-root.css-18l5dsu div input`,
+  searchDropdown: `[data-popper-placement="bottom"]`,
+  dropdownString: name => `[role="presentation"]:has-text('${name}')`,
+};
+export default class MainPage extends BasePage {
+  get searchBar() {
+    return new InputField(this.page, SELECTORS.searchBar);
+  }
+
+  async search(text: string) {
+    await this.searchBar.type(text);
+    await this.locator(SELECTORS.searchDropdown).isVisible();
+  }
+
+  async isListItemVisible(name: string): Promise<boolean> {
+    await this.page.locator(SELECTORS.dropdownString(name)).waitFor({ state: 'visible' });
```

Comment:
This method was already implemented by us earlier in `ui/elements/list.ts`.

```
// in file: ui/pages/catalog/catalog-page.ts

  get resultsList() {
    return new List(this.page, SELECTORS.resultList, SELECTORS.listItem);
  }
```
Then you can call it like this:  `this.resultsList.isListItemVisible()`